examples/qs-only/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "@algolia/qs-only",
+  "description": "Auto complete query suggestion Sandbox",
+  "version": "1.0.0-alpha.43",
+  "private": true,
+  "license": "MIT",
+  "main": "index.html",
+  "scripts": {
+    "build": "parcel build index.html",
+    "start": "parcel index.html"
+  },
+  "dependencies": {
+    "@algolia/autocomplete-js": "1.0.0-alpha.43",
+    "@algolia/autocomplete-plugin-algolia-insights": "1.0.0-alpha.43",
```

Comment:
We can remove all the unused dependencies.

Reply:
Fixed.